tests/foreman/cli/test_http_proxy.py

```diff
@@ -293,6 +300,13 @@ def test_positive_assign_http_proxy_to_products(module_org, module_target_sat):
         {'ids': f"{product_a['id']},{product_b['id']}", 'http-proxy-policy': 'none'}
     )
     assert 'Product proxy updated' in res
+    module_target_sat.wait_for_tasks(
```

Comment:
I beleive this function returns `List of sat.api.ForemanTask entities`, would it be possible to use assersion on return value for this call?

Reply:
Yup it returns a list of found tasks. However, since they are read before the pull loop, some of them are pending. So I'm fine with the subsequential assertion...